src/components/SidebarComponent.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import styles from '../assets/stylesheets/sidebar.module.css';
+
+function Sidebar() {
+  return (
+    <div className={styles.sidebar}>
+      <ul>
+        <li>Home</li>
+        <li>About</li>
+        <li>Services</li>
+        <li>Contact</li>
+      </ul>
```

Comment:
And, about this menu options, they don't correspond to the project's requirements. Check this: In the navigation panel, the user can see links to:
Motorcycles/doctors/classes/items that you selected as a theme, in our case **Coworking spaces**.
"**Reserve**" form.
"**My reservations**".
"**Add** motorcycle/doctor/class/item that you selected as a theme" (the link is visible to everybody).
"**Delete** motorcycle/doctor/class/item that you selected as a theme" (the link is visible to everybody).